### PR TITLE
Some proposed changes to Livepeer Explorer landing

### PR DESCRIPTION
**Pull request by chrishobcroft**
Removed the video tutorial, edit some copy, changed some routing.

**What does this pull request do? Explain your changes. (required)**

It makes some changes to explorer.livepeer.org landing page, to help new explorers in Livepeer to begin to engage with the protocol more easily.

Specifically, it promotes the section where a user can enter any Ethereum address (including their own), to begin to explore livepeer's protocol.

It also rewords some copy, to make it more accessible to a wider audience - the 2.6 million who have no idea about bonding and inflationary reward yet - but they will soon, maybe!

**Specific updates (required)**

- Remove the video (for now, pending a new video to be put in it's place to help new entrants joined the community)

- Rebrand "Explorer Protocol Activity" to "Explore Your Account"

   - Adapt the copy to make it friendlier, and easier to engage with. Also, to seek to personalise it somewhat, and make the user feel at the centre of this.

- Rebrand "Play the Delegation Game" to "BOND your LPT to a Node"

   - Someone should totally gamify this, but not us. At least, not yet.
   - For now, I propose to lead them down the garden path, to the specific call-to-action that we want them to take. To BOND.
   - GET TOKEN is finished now. Remove from everywhere, let's tidy this up :)

**How did you test each of these updates (required)**

I tested them in my mind's eye, tweaked some text about, fit @jozanza's pattern. It'll be fine. Can we push them to staging first and then have a look?

**Does this pull request close any open issues?**

No.

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/explorer/src/views/Landing/index.js [hunk 1]
```diff
@@ -22,19 +22,15 @@ const Landing = ({ history, ...props }) => (
...
         <br />
-        <Video src="/static/media/landing.mp4" />
-        <br />
-        <br />
         <div style={{ display: 'flex' }}>
           <InfoBox>
             <h2>
               <Icon use="explore" />&nbsp;
-              {`Explore Protocol Activity`}
+              {`Explore Your Account`}
```
Comment (Randy1Burrell) [inline packages/explorer/src/views/Landing/index.js:29]: @chrishobcroft Great job. Just a few comments/questions.

> Wouldn't it be better to say "Explore an account" ?
- Reasoning: A user will know that they can also view others accounts. Ex. friends' account number they have and might be able to inform that friend of current unbonded tokens they don't know about.
- The description would make it clear that logging into a web3 enabled wallet is not mandatory for viewing an account.

File: packages/explorer/src/views/Landing/index.js [hunk 2]
```diff
@@ -50,7 +46,7 @@ const Landing = ({ history, ...props }) => (
                 name="address"
                 type="search"
                 pattern="^0x[a-fA-F0-9]{40}$"
-                placeholder="Enter an ETH account address"
+                placeholder="Enter your ETH account address"
```
Comment (Randy1Burrell) [inline packages/explorer/src/views/Landing/index.js:49]: @chrishobcroft same question as above.

File: packages/explorer/src/views/Landing/index.js (hunk 1 above)
Comment (chrishobcroft) [reply packages/explorer/src/views/Landing/index.js:29]: Hmm. Well. Good question @Randy1Burrell.

When I think about priorities, I compare the following objectives:

**- Objective 1:** Helping a token holder become aware that they have LPT

vs.

**- Objective 2:** Helping a token holder to check whether their friend has LPT

For me, Objective 1 is higher priority, as it is the shortest route for any keyholder to check whether they have LPT themselves.

Objective 2 can also be met using this process, but is more likely to happen once a token holder has established that _they_ have LPT in the first place, and hence should be lower priority.

What do you think?

File: packages/explorer/src/views/Landing/index.js (hunk 2 above)
Comment (chrishobcroft) [reply packages/explorer/src/views/Landing/index.js:49]: Same answer as above.

**Review by Randy1Burrell — APPROVED** [review]

**Review by Randy1Burrell — APPROVED** [review]
Merging this but it will be overwritten as soon as frontend-modification features are merged.

**Comment by Randy1Burrell** [comment]
Thanks @chrishobcroft for contributing this.

**Pull request merged**